### ISSUE TITLE
"master, chief" being added to prompts for AOM3 [d124fc18f0]

### ISSUE THREAD

**Issue by Gazzoo-byte** [mix1009/model-keyword#30]
As above, somehow [this model](https://huggingface.co/WarriorMama777/OrangeMixs/blob/main/Models/AbyssOrangeMix3/AOM3.safetensors) is getting "master, chief" added, no custom mappings etc

**Comment by Aryetis** [comment]
Yup I've had all sort of tokens added in my prompts recently without explanations. For instance using this image in img2img :
![06663-1637033246-girl soulworker anime stella unibell standing simple background smile](https://user-images.githubusercontent.com/4246001/232122228-d667073c-d2fb-40b3-b3fb-a713d72ab030.png)

It adds "girl" in front of the prompt.

I've also seen some kanjis and other random words being added from time to time in txt2img.

I'm turning this extension off for now.
